src/components/NewApeAvatar/NewApeAvatar.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```

Comment:
is this still a WIP? it looks similar enough to ApeAvatar that maybe it would be better to change that to be able to handle simple props instead of user/profile objects?

Reply:
My only issue leaving this here is scope. I just created a ticket here https://github.com/coordinape/coordinape/issues/545 that specifies what should be done. I might finish later today.